wolfcrypt/src/fe_low_mem.c

```diff
@@ -55,7 +55,11 @@ void lm_copy(byte* x, const byte* a)
         x[i] = a[i];
 }
 
-#ifndef FREESCALE_LTC_ECC
+#if ((defined(HAVE_CURVE25519) && !defined(CURVE25519_SMALL)) || \
+	 (defined(HAVE_ED25519) && !defined(ED25519_SMALL))) &&      \
+	!defined(FREESCALE_LTC_ECC)
+	/* to be Complementary to fe_low_mem.c */
+#else
```

Comment:
Can you clarify the reasoning behind this?  Was it because CURVE25519_SMALL was defined, but not ED25519_SMALL?

This also seems to change the behavior when FREESCALE_LTC_ECC is defined.  Before, if FREESCALE_LTC_ECC was defined, we did not compile in the fp_init() stub below.  Now, with these changes, when FREESCALE_LTC_ECC is defined, we do compile in the fp_init() stub.

Reply:
Thanks for pointing out. Rollbacked #ifndef FREESCALE_LTC_ECC.

The #if condition became a bit redundant, now. But thought it would be simpler
to keep the same condition in fe_operation.c and put fe_init in #else clause
to making it complementary each other.

Reply:
Thanks for talking about this today.  As discussed, Kojo will roll this commit back to the original one, and switch the code comment to read "to be Complementary to fe_operations.c".